docs/csharp/csharp-7.md

```diff
@@ -69,6 +69,10 @@ The most common use for this feature will be the `Try` pattern. In this
 pattern, a method returns a `bool` indicating success or failure and an
 `out` variable that provides the result if the method succeeds.
 
+> [!NOTE]
+> When using the `out` variable declaration, the scope of the declared variable is the same as it was in previous 
+> versions of C#. In other words, this is syntax sugar that enables inlining.  
```

Comment:
> the scope of the declared variable is the same as it was in previous versions of C#

I find this wording confusing: previous versions of C# did not support `out` variables, so how could the scope be the same?

> In other words, this is syntax sugar that enables inlining.  

I'm also not sure it's clear what "inlining" means here, or how does that explain the scoping rules.

I would probably include an example and phrase it something like this:

> When using the `out` variable declaration, the declared variable "leaks" into the outer scope of the `if` statement. This allows you to use the variable afterwards:
>
> ```c#
> if (!int.TryParse(input, out int result))
>     return null;
> 
> return result;
> ```